bot/exts/evergreen/cowsay.py

```diff
@@ -0,0 +1,57 @@
+from random import choice
+
+from cowsay import get_output_string
+from discord import Embed
+from discord.ext import commands
+
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+
+class Cowsay(commands.Cog):
+    """Cog for the cowsay command."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["cow"])
+    async def cowsay(self, ctx: commands.Context, character: str = "Cow", *, text: str = "I'm a Cow") -> None:
+        """
+        Generates some cowsay ASCII art and sends it in Discord.
+
+        There are multiple character options to choose from. Here is a list:
+
+        Cow
+        Trex
+        Dragon
+        Cheese
+        Beavis
+        Daemon
+        Ghostbusters
+        Kitty
+        Meow
+        Milk
+        Pig
+        Stegosaurus
+        Stimpy
+        Turkey
+        Turtle
+        Tux
+        """
+        text = text.lower()
+        character = character.lower()
+        try:
+            msgbody = get_output_string(character, text)
+        except BaseException:
```

Comment:
This shouldn't be a bare except. Can you except a function that would be raise if there was an invalid character?

Reply:
sure

Reply:
what would you suggest?

Reply:
Try sending an "invalid character", and if it errors, you should except that error. If it doesn't error (which I think it might not), then there will be no need to catch the error.

Reply:
fixed in 2a58874